dotCMS/src/main/java/com/dotcms/rendering/velocity/services/ContainerLoader.java

```diff
@@ -241,7 +241,7 @@ private InputStream buildVelocity(Container container, String uuid, PageMode mod
                     .append(" data-dot-title=")
                     .append("\"$UtilMethods.javaScriptify($ContentletTitle)\"")
                     .append(" data-dot-can-edit=")
-                    .append("\"$contents.doesUserHasPermission($CONTENT_INODE, 2, true)\"")
+                    .append("\"$contents.doesUserHasPermission($CONTENT_INODE, 2, false)\"")
```

Comment:
I think for this, has been introduced the: PageMode.get(request).respectAnonPerms
check it out